dcm4che-tool/dcm4che-tool-jpg2dcm/src/main/java/org/dcm4che3/tool/jpg2dcm/Jpg2Dcm.java

```diff
@@ -146,6 +154,10 @@ public static void main(String[] args) {
             long fin = System.currentTimeMillis();
             LOG.info("Encapsulated " + jpg2Dcm.pixelDataFile + " to " + dcmFile.getPath() + " in "
                     + (fin - start) + "ms.");
+        } catch (ParseException e) {
+            System.err.println("jpg2dcm: " + e.getMessage());
+            System.err.println(rb.getString("try"));
+            System.exit(2);
         } catch (Exception e) {
             LOG.error("Error: \n", e);
             e.printStackTrace();
```

Comment:
Hi @vrindanayak, just one last small thing:
On this generic exception the tool should also exit with a non-zero exit status, to indicate that there was an error. This is important if the tool is used within a script.
Just add a `System.exit(2);`.